CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [0.7.0 UNRELEASED]
+## 0.7.0
```

Comment:
```suggestion
## [0.7.0]
```
Earlier, we haven't removed the square brackets, but maybe we should?

Reply:
I think we should stay consistent and not remove brackets

Reply:
That is straight up an error from me!